spec/models/spree/product_spec.rb

```diff
@@ -10,6 +10,10 @@ module Spree
 
       context '#duplicate' do
         it 'duplicates product' do
+          variant = product.variants.first
+          variant.supplier = create(:supplier_enterprise)
+          variant.save!
+
```

Comment:
Doesn't the product factory create the variant with a supplier? If not, I think it should. 
In any case, I don't think this test should need to deal with suppliers at all.

Reply:
The factory does create the supplier, I am not sure why this is there. Removed !